ext/node/polyfills/internal/crypto/hash.ts

```diff
@@ -133,6 +133,15 @@ export class Hash extends Transform {
         return Buffer.from(digest).toString(encoding);
     }
   }
+
+  /**
+   * Get the list of implemented hash algorithms.
+   * @returns Array of hash algorithm names.
+   */
+
+  static getHashes() {
```

Comment:
`crypto.Hash` class doesn't seem having this static method. Let's not add this?

```shellsession
$ node
Welcome to Node.js v18.14.1.
Type ".help" for more information.
> crypto.Hash.getHashes
undefined
```

Reply:
Sure!